app/kategori/[...slug]/page.tsx

```diff
@@ -0,0 +1,34 @@
+import { notFound } from "next/navigation"
+import React from "react"
+
+import loadCategoryPage from "@/app/kategori/[...slug]/loadCategoryPage"
+import CategoryPageLayout from "@/components/pages/categoryPageLayout/CategoryPageLayout"
+import { NodeGoCategory } from "@/lib/graphql/generated/dpl-cms/graphql"
+
+async function page(props: { params: Promise<{ slug: string[] }> }) {
+  const params = await props.params
+  const { slug } = params
+  const slugString = slug.join("/")
+  const data = await loadCategoryPage(slugString)
+  const routeType = data.route?.__typename
+
+  if (routeType === "RouteRedirect") {
+    // TODO: implement redirect
```

Comment:
Should we make tasks for these TODOs? It could be great to not forget about 'em, cuz they're quite important(?)